### PR TITLE
{bio}[foss/2017b,intel/2017b] cutadapt v1.16

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/1e475f8f3e3a844eb04478c1a767d650 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/919e9d6c3d66a4905e45cbeacf15b5ea for a full test report.

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.16-foss-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+# Modufied by: Albert Bogdanowicz
```
Comment (verdurin) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.16-foss-2017b-Python-2.7.14.eb]: @vanzod Not your doing but please fix the typo here

**Comment by smoors** [comment]
Going in, thanks @vanzod!

**Comment by verdurin** [comment]
Going in, thanks @vanzod!

**Pull request merged**